chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/assaults/ExceptionAssault.java

```diff
@@ -26,7 +26,7 @@
 /** @author Thorsten Deelmann */
 public class ExceptionAssault implements ChaosMonkeyRequestAssault {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(ExceptionAssault.class);
+  private static final Logger Logger = LoggerFactory.getLogger(ExceptionAssault.class);
```

Comment:
Actually we should use Lombok to get the logger, but we can do that on another PR.